integration/up/hybridmode_test.go

```diff
@@ -49,9 +61,25 @@ dev:
 
 	svcDockerfile = `FROM busybox
 ENV ENV_IN_IMAGE value_from_image`
```

Comment:
if I change Dockerfile env key to `ANOTHER_ENV` e2e test still passes. Could be that some envs from dockerfile or pod are not checked properly in the script?

Reply:
As we discussed offline, this seems to be a bug in how we calculate the digest for rebuilding the dev container image: https://github.com/okteto/okteto/issues/3871